tests/cli/mirroring_smoke_tests/e2e/fluvio-core.bats

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bats
+
+TEST_HELPER_DIR="$BATS_TEST_DIRNAME/../../test_helper"
+export TEST_HELPER_DIR
+
+load "$TEST_HELPER_DIR"/tools_check.bash
+load "$TEST_HELPER_DIR"/fluvio_dev.bash
+load "$TEST_HELPER_DIR"/bats-support/load.bash
+load "$TEST_HELPER_DIR"/bats-assert/load.bash
+
+setup_file() {
+    CURRENT_DATE=$(date +%Y-%m)
+    export CURRENT_DATE
+
+    REMOTE_PROFILE=local
+    export REMOTE_PROFILE
+    debug_msg "Remote profile: $REMOTE_PROFILE"
+
+    REMOTE_NAME=edge1
+    export REMOTE_NAME
+    debug_msg "Remote name: $REMOTE_NAME"
+
+    REMOTE_PROFILE_2=local2
+    export REMOTE_PROFILE_2
+    debug_msg "Remote profile: $REMOTE_PROFILE_2"
+
+    REMOTE_NAME_2=edge2
+    export REMOTE_NAME_2
+    debug_msg "Remote name: $REMOTE_NAME_2"
+
+    MESSAGE="$(random_string 7)"
+    export MESSAGE
+    debug_msg "$MESSAGE"
+
+    TOPIC_NAME=mirror-topic
+    export TOPIC_NAME
+    debug_msg "Topic name: $TOPIC_NAME"
+
+    HOME_PROFILE=$($FLUVIO_BIN profile)
+    export HOME_PROFILE
+    debug_msg "Home profile: $HOME_PROFILE"
+
+    HOME_NAME="home"
+    export HOME_NAME
+    debug_msg "Home name: $HOME_NAME"
+}
+
+@test "Can register first local remote cluster" {
+    run timeout 15s "$FLUVIO_BIN" remote register "$REMOTE_NAME"
+
+    assert_output "remote cluster \"$REMOTE_NAME\" was registered"
+    assert_success
+}
+
+@test "Can register second local remote cluster" {
+    run timeout 15s "$FLUVIO_BIN" remote register "$REMOTE_NAME_2"
+
+    assert_output "remote cluster \"$REMOTE_NAME_2\" was registered"
+    assert_success
+}
+
+@test "Export home to remote 1" {
+    run timeout 15s "$FLUVIO_BIN" remote export "$REMOTE_NAME" --file remote.json
+    assert_output ""
+    assert_success
+
+    run jq -r .home.id remote.json 
+    assert_output "$HOME_NAME"
+    assert_success
+
+    run jq -r .home.remoteId remote.json 
+    assert_output "$REMOTE_NAME"
+    assert_success
+
+    run jq -r .home.publicEndpoint remote.json 
+    assert_output --partial ":30003"
```

Comment:
where is this come from? maybe this should be specified in the set up?

Reply:
It's the default public port for Kubernetes clusters. I change to use a env var